<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="NameLayout">
    <item>
     <widget class="QLabel" name="NameLabel">
      <property name="text">
       <string>Username:               </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="NameTextField"/>
    </item>
    <item>
     <widget class="QLabel" name="NameLabelCheck">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;* Already Existing&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="PassLayout">
    <item>
     <widget class="QLabel" name="PassLabel">
      <property name="text">
       <string>Password:                </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="PassTextField"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="RePassLayout">
    <item>
     <widget class="QLabel" name="RePassLabel">
      <property name="text">
       <string>Re-Type Password:  </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="RePassTextField"/>
    </item>
    <item>
     <widget class="QLabel" name="RePassLabelCheck">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;* Already Existing&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="BirthDateBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>351</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Date of Birth</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="MonthLayout">
     <item>
      <widget class="QLabel" name="MonthLabel">
       <property name="text">
        <string>Month</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxMonth">
       <item>
        <property name="text">
         <string>Jan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Feb</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>April</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>May</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>June</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>July</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Aug</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sept</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Oct</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nov</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dec</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>101</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="DayLayout">
     <item>
      <widget class="QLabel" name="DayLabel">
       <property name="text">
        <string>Day</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NumberTextField"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>111</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="YearLayout">
     <item>
      <widget class="QLabel" name="YearLabel">
       <property name="text">
        <string>Year</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="YearTextField"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="LabelAgeCheck">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff1500;&quot;&gt;* Age is less than 12&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="GenderBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Gender</string>
   </property>
   <widget class="QRadioButton" name="maleRadio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="femaleRadio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="AccountTypeBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Account Type</string>
   </property>
   <widget class="QRadioButton" name="UserRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AdminRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GenresBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>381</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Favourite Genre(s)</string>
   </property>
   <widget class="QCheckBox" name="ActionCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Action</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ComedyCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Comedy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="RomanceCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Romance</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DramaCheckBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Drama</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="HorrorCheckBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Horror</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="OtherCheckBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Other</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="RegisterButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="AllFieldCheckerLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>510</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;* All fields must be filled&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
